File: RivoltaAlpha/count_zeros.py
def is_power_of_two(n: int) -> bool:
    if n <= 0:
        return False
    return (n & (n - 1)) == 0

# Example usage:
print(is_power_of_two(8))  # Output: True
print(is_power_of_two(6))  # Output: False

File: Leader254/count_zeros.js
function isPowerOfTwo(int) {
  if (int <= 0) return false;

  while (int % 2 === 0) {
    int = int / 2;
  }
  return int === 1;
}

console.log(isPowerOfTwo(8)); // true
console.log(isPowerOfTwo(6)); // false
